docbrown/core/src/tgraph_shard.rs

```diff
@@ -496,7 +497,7 @@ mod temporal_graph_partition_test {
         let g = TGraphShard::default();
 
         for (v, (t_start, _)) in intervals.0.iter().enumerate() {
-            g.add_vertex(*t_start, v.try_into().unwrap(), &vec![])
+            g.add_vertex(*t_start, v as u64, &vec![])
```

Comment:
what is this? why is this?

Reply:
its always a u64 for this test anyway, because it complains about the into type